content/source/docs/enterprise/sentinel/import/index.html.md

```diff
@@ -17,6 +17,18 @@ state and plan.
 - [tfconfig](./tfconfig.html) - This provides access to a Terraform configuration, the set of "tf" files that are used to describe the desired infrastructure state.
 - [tfstate](./tfstate.html) - This provides access to the Terraform state, the file used by Terraform to map real world resources to your configuration.
 
+Terraform Enterprise allows you to create mocks of these imports from executable
```

Comment:
I think this is the first mention of "executable plans" I've ever seen. What is it? It sounds like it might mean either a .tfplan binary plan file, or a TFE plan entity (json-api type `plans`) that isn't marked as speculative. I'm guessing it's the former?

Reply:
It's the .tfplan binary file, i didn't realize that wasn't really an external term. I can change this!